documentation/docs/13-page-options.md

```diff
@@ -70,6 +70,15 @@ For _pages_, we skirt around this problem by writing `foo/index.html` instead of
 
 Note that this will disable client-side routing for any navigation from this page, regardless of whether the router is already active.
 
+#### Troubleshooting
+
+If you encounter an error like 'Route ... is marked as prerenderable, but was not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't actually prerendered.
```

Comment:
Out of scope for this PR, but I think a dedicated page that lists all errors thrown by Kit would be nice. The errors thrown by Kit would then also have a link pointing to that site where you can read more about that error.